src/test/java/api/loans/DueDateNotRealTimeScheduledNoticesProcessingTests.java

```diff
@@ -37,12 +35,19 @@
 import io.vertx.core.json.JsonObject;
 
 public class DueDateNotRealTimeScheduledNoticesProcessingTests extends APITests {
+
+  private final static UUID TEMPLATE_ID = UUID.randomUUID();
```

Comment:
Why is there a need for a global constant for all of the tests?

Reply:
After adding template validation, the tests fail without template. The template that I added to templateClient should have the same id as notice.

Reply:
Can the template be created during the test and then the id used when creating the notice?

Reply:
But it means that we should call templateFixture.createDummyNoticeTemplate for each test with the same ID also.  Is it OK?
I extracted it to setUp method with @Before annotation to avoid duplication.